wpeverest/everest-forms
# Tweak - Remove rating request notification

### All Submissions:


### Changes proposed in this Pull Request:

This PR removes the rating request notification from the builder page.

### How to test the changes in this Pull Request:

1. Goto All forms.
2. Edit a form
3. Verify the rating request notification.

### Types of changes:



### Other information:


### Changelog entry

> Tweak - Remove rating request notification.
